### ISSUE TITLE
QuickJS Error when trying to get World Space for raycasting

### ISSUE THREAD

**Issue by domis4** [Geequlim/ECMAScript#14]
Hello, first of thanks for this great implementation of ES for Godot. It helps me alot getting started.

i ran into an issue when trying to get the world space state, in order to cast an ray between two Spatials. (i want to copy the Y position of the element below the player)

```js
export default class Example extends godot.KinematicBody {
  _physics_process(delta) {
    var world = this.get_world()
    var spaceState = world.get_direct_space_state() // this line throws the error
  }
}
```

```
E 0:00:01.516   variant_to_var: Parameter ' data->ecma_object ' is null.
  <C++-Quellcode>modules/ECMAScript/quickjs/quickjs_binder.cpp:132 @ variant_to_var()
```

This method of ray casting is recommendet in http://docs.godotengine.org/en/3.2/tutorials/physics/ray-casting.html

My Godot Built is based on
- Godot `e707b712e8` (3.2 branch)
- ECMAScript `ec69d88` (maybe a bit outdated)

**Comment by domis4** [comment]
@Geequlim may i be able to support you? :) maybe you can give me a hint where why this issue happens.

**Comment by Geequlim** [comment]
Thanks for your reporting.
This issue is fixed by 35fe07f.
Happy coding!
